File: tests/fixtures/positive/security/WeakKeySize.java
package test;

import java.security.KeyPairGenerator;
import javax.crypto.KeyGenerator;

class Crypto {
    void keys() throws Exception {
        KeyPairGenerator rsa = KeyPairGenerator.getInstance("RSA");
        rsa.initialize(1024);
        KeyGenerator aes = KeyGenerator.getInstance("AES");
        aes.init(64);
    }
}

File: tests/fixtures/positive/security/WeakKeySize.kt
package test

import java.security.KeyPairGenerator
import javax.crypto.KeyGenerator

class Crypto {
    fun keys() {
        val rsa = KeyPairGenerator.getInstance("RSA")
        rsa.initialize(1024)
        val aes = KeyGenerator.getInstance("AES")
        aes.init(64)
    }
}
